### PR TITLE
[17.0][MIG] sale_order_invoicing_grouping_criteria

### PR DESCRIPTION
**Pull request by gbrito**

### CODE REVIEW

File: sale_order_invoicing_grouping_criteria/tests/test_sale_order_invoicing_group_criteria.py [hunk 1]
```diff
@@ -92,6 +92,7 @@ def test_invoicing_grouping_partner_criteria_as_demo(self):
         user_demo.groups_id = [
             (4, self.env.ref("sales_team.group_sale_salesman_all_leads").id)
         ]
```
Comment (CRogos) [inline sale_order_invoicing_grouping_criteria/tests/test_sale_order_invoicing_group_criteria.py:94]: Is this still needed or overwritten by the line below?

**Review by CRogos — COMMENTED** [review]
Could you also cleanup the git history? Add the proper naming to the last, and squash the yellow commits.

https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate


image

**Comment by HaraldPanten** [comment]
Hi @gbrito Will you continue with this PR?  THX!

**Comment by HaraldPanten** [comment]
Closing, as it has been superseded

**Pull request closed without merging**